build.gradle

```diff
@@ -103,10 +103,6 @@ subprojects {
  * in a branch if there are only betas and rcs in the branch so we have
  * *something* to test against. */
 VersionCollection versions = new VersionCollection(file('server/src/main/java/org/elasticsearch/Version.java').readLines('UTF-8'))
-if (versions.currentVersion != VersionProperties.elasticsearch) {
```

Comment:
The check is moved into version collections.
I tried to move checks and decisions there for better visibility.